extenders/Message.js

```diff
@@ -1,30 +1,28 @@
 const { Structures } = require("discord.js");
 
-module.exports = Structures.extend("Message", Message => {
-  return class extends Message {
+module.exports = Structures.extend("Message", Message => class extends Message {
 
-    constructor(...args) {
-      super(...args);
+  constructor(...args) {
+    super(...args);
 
-      this.settings = this.guild ? this.client.getSettings(this.guild.id) : this.client.settings.get("default");
-      
-      this.flags = [];
-    }
+    this.settings = this.guild ? this.client.getSettings(this.guild.id) : this.client.settings.get("default");
 
-    response(emoji = "❌", content, embed, options = {}) { // eslint-disable-line no-unused-vars
-      return this.channel.send(`${this.author} \`|${emoji}|\` ${content}`, embed);
-    }
+    this.flags = [];
+  }
+
+  response(emoji = "❌", content, embed, options = {}) { // eslint-disable-line no-unused-vars
```

Comment:
I'm pretty sure you can get rid of this.

Reply:
its used by way to many places

Reply:
So it is... I thought it was hardly used lol my bad